source/ZividNETSamples.sln

```diff
@@ -31,12 +31,14 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CaptureFromFileVis3D", "App
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CaptureVis3D", "Applications\Basic\Visualization\CaptureVis3D\CaptureVis3D.csproj", "{68EB8781-17AA-42E9-9524-0C18A45278C3}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HandEyeCalibration", "Applications\Advanced\HandEyeCalibration\HandEyeCalibration.csproj", "{8ACFFEC6-34E5-4E14-B08C-60051B1CFD2F}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HandEyeCalibration", "Applications\Advanced\HandEyeCalibration\HandEyeCalibration\HandEyeCalibration.csproj", "{8ACFFEC6-34E5-4E14-B08C-60051B1CFD2F}"
```

Comment:
It looks like this was changed in the commit which adds `UtilizeEyeInHandCalibration`, rather than in the one that moves `HandEyeCalibration`. So the commit that moves `HandEyeCalibration` probably doesn't build now, you should check that.

Reply:
Done. Moved it to the commit that moves HE sample